src/System.IO.Compression.Brotli/System/Interop/Library.OSX.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.IO.Compression
+{
+    internal static partial class Library
```

Comment:
I think we can with using define constants, but to be convenient I did it like in corefx https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Windows/Interop.Libraries.cs (and the same for unix and osx)